test/development/acceptance-app/undefined-default-export.test.ts

```diff
@@ -5,6 +5,7 @@ import { sandbox } from 'development-sandbox'
 describe('Undefined default export', () => {
   const { next } = nextTestSetup({
     files: new FileRef(path.join(__dirname, 'fixtures', 'default-template')),
+    patchFileDelay: 250,
```

Comment:
These delays will be replaced by proper change detection (per PR description) - but that will come later